x-pack/elastic-agent/magefile.go

```diff
@@ -456,6 +456,19 @@ func CrossBuildGoDaemon() error {
 	return devtools.CrossBuildGoDaemon()
 }
 
+// PackageAgentCore cross-builds and packages distribution artifacts containing
+// only elastic-agent binaries with no extra files or dependencies.
+func PackageAgentCore() {
+	start := time.Now()
+	defer func() { fmt.Println("packageAgentCore ran for", time.Since(start)) }()
+
+	mg.Deps(CrossBuild, CrossBuildGoDaemon)
```

Comment:
I don't think that GoDaemon is included in the core package (at least it's not in the 8.X versions) and it's needed only by the rpm or deb full agent packages so I think we can skip it here and save some time and money ;)

Reply:
It came from the elastic-agent main branch as it is :)

Reply:
Then maybe we have to get rid of it also on the elastic-agent main... What I know for sure is that the `elastic-agent-core` DRA do not include `god` executable in the package, that's why I was thinking of getting rid of the dependency for this target ;)

I created an issue to track this https://github.com/elastic/elastic-agent/issues/4467 and check if we can get rid of this also from elastic-agent main branch :boom:

Reply:
Done and it works :)